cloud-gate.spec

```diff
@@ -1,5 +1,5 @@
 Name:           cloud-gate
-Version:        1.3.1
+Version:        1.3.2
```

Comment:
The `master` branch already has v `1.3.2`.

Reply:
yes  I need to rebase this one